File: ace75/productExceptSelf.py
from typing import List

class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        res = []
        product = 1
        for i in range(len(nums)):
            product = 1
            for j in range(len(nums)):
                if i == j:
                    continue
                product *= nums[j]
            res.append(product)
        return res

File: ace75/productExceptSelf.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        std::vector<int> res;
        int product = 1;
        for(int i=0;i<nums.size();i++){
            product = 1;
            for(int j=0;j<nums.size();j++){
                if(i==j){
                    continue;
                }
                product*=nums[j];
            }
            res.push_back(product);
        }
        return res;
    }
};